// try/catch

try {
    console.log('normal');
} catch {
    console.log('error');
}

